### PR TITLE
Add About us page

### PR DESCRIPTION
**Pull request by fullonic**
## Description

Please describe your pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. When linking to an issue, please use `refs #...` in the description of the pull request.

Partial solves issue #22 

Adds endpoint:
`/api/about-us`

Adds url (html page):
`/about-us`

### CODE REVIEW

File: jobsapp/api/views/common.py [hunk 1]
```diff
@@ -27,3 +34,18 @@ def get_queryset(self) -> "QuerySet[Job]":
...
+    def get(self, request: Request, *args: Any, **kwargs: Any) -> Response:
+        about_content = FlatPage.objects.filter(url="/about-us/").first()
+        # REVIEW: Is this response format okay?
+        content = {
+            "url": reverse("about-us"),
+            "title": _(about_content.title),
+            # REVIEW: If there is any html tag we will send it, should we remove it before?
+            "content": _(about_content.content),
+        }
+        return Response(data=content, status=status.HTTP_200_OK)
```
Comment (francescarpi) [inline jobsapp/api/views/common.py:51]: In my opinion, is not necessary to return the "url". And at the moment I don't know how manage translations for flatpages. 

At the moment, no return title and content with gettext.

**Review by francescarpi — COMMENTED** [review]
Hi! Good job.

We also need the Django view.

Thank you!

File: jobsapp/api/views/common.py (hunk 1 above)
Comment (fullonic) [reply jobsapp/api/views/common.py:51]: >  Django view?

Hey @francescarpi , there is already a django view for that. Is managed by flatpages it self. We can add or edit content on the admin page. I'm removing the translation part and I can take a look on _how translate flatpages_ later.

Comment (francescarpi) [reply jobsapp/api/views/common.py:51]: Ok, thanks!

**Comment by francescarpi** [comment]
Sorry, other PR has been accepted before your. Could you resolve conflicts, pls?

Thanks!

**Comment by fullonic** [comment]
@francescarpi done! Thanks :+1:

**Comment by francescarpi** [comment]
Black does not pass. 
Thanks

**Pull request merged**